pkg/meta/redis.go

```diff
@@ -3495,8 +3495,12 @@ func (m *redisMeta) checkServerConfig() {
 		}
 	}
 	start := time.Now()
-	_ = m.rdb.Ping(Background)
-	logger.Infof("Ping redis: %s", time.Since(start))
+	_, err = m.rdb.Ping(Background).Result()
+	if err != nil {
+		logger.Errorf("Ping error: %s", err.Error())
```

Comment:
show address of redis, or just `ping redis: %s`

Reply:
😁 done